packages/orchestrator-ui-components/src/icons/WfoStatistic.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FC } from 'react';
+import { WfoIconProps } from './WfoIconProps';
+
+export const WfoStatistic: FC<WfoIconProps> = ({
+    width = 20,
+    height = 20,
+    color = '#000000',
+}) => (
+    <svg
+        width={width}
+        height={height}
+        viewBox="0 0 20 20"
+        version="1.1"
+        color={color}
+        xmlns="http://www.w3.org/2000/svg"
+        className="wfoStatisticIcon"
+    >
+        <title>icon/statistic</title>
+        <g xmlns="http://www.w3.org/2000/svg">
+            <path
+                fill={color}
+                fill-rule="nonzero"
```

Comment:
Didn't this throw an error? Copying from an SVG file to a React component means you need to replace `fill-rule` with `fillRule` and similar

Reply:
will be checked